Gems/Prefab/PrefabBuilder/PrefabGroup/ProceduralAssetHandler.cpp

```diff
@@ -68,6 +79,84 @@ namespace AZ::Prefab
         extensions.push_back(PrefabGroupAssetHandler::s_Extension);
     }
 
+    void PrefabGroupAssetHandler::AssetTypeInfoHandler::AddContextMenuActions(
+        [[maybe_unused]] QWidget* caller,
+        QMenu* menu,
+        const AZStd::vector<AzToolsFramework::AssetBrowser::AssetBrowserEntry*>& entries)
+    {
+        using namespace AzToolsFramework::AssetBrowser;
+        auto entryIt = AZStd::find_if
+        (
+            entries.begin(),
+            entries.end(),
+            [](const AssetBrowserEntry* entry) -> bool
+            {
+                return entry->GetEntryType() == AssetBrowserEntry::AssetEntryType::Product;
+            }
+        );
+
+        if (entryIt == entries.end())
+        {
+            return;
+        }
+        else  if ((*entryIt)->GetEntryType() == AssetBrowserEntry::AssetEntryType::Product)
+        {
+            ProductAssetBrowserEntry* product = azrtti_cast<ProductAssetBrowserEntry*>(*entryIt);
+            if (product->GetAssetType() == azrtti_typeid<ProceduralPrefabAsset>())
+            {
+                AZ::Data::AssetId assetId = product->GetAssetId();
+                menu->addAction("Save as Prefab...", [assetId, this]()
+                {
+                    QString filePath = AzQtComponents::FileDialog::GetSaveFileName(nullptr, QString("Save to file"), "", QString("Prefab file (*.prefab)"));
+                    if (filePath.isEmpty())
+                    {
+                        return;
+                    }
+                    if (SaveAsAuthoredPrefab(assetId, filePath.toUtf8().data()))
+                    {
+                        AZ_Printf("Prefab", "Prefab was saved to a .prefab file %s", filePath.toUtf8().data());
```

Comment:
Should this say "Procedural Prefab was saved to .prefab file %s"?

Reply:
This saves a procedural prefab as an authored Prefab, so the Prefab was saved to the file.

Reply:
Right, I just meant the message sounds like an authored prefab was saved (I think most people will assume 'prefab' = 'authored prefab').  Its just a debug message though so not a big deal